content/webapp/pages/covid-book-your-ticket.tsx

```diff
@@ -200,21 +194,27 @@ const CovidWeAreOpenPage = () => {
                     you need it, as there is limited availability to allow for
                     social distancing.
                   </p>
-                  <CovidH4>Ordering material</CovidH4>
                   <p>
                     <ul className="no-margin">
                       <li>
                         All your online requests will be delivered to the Rare
-                        Materials Room. To start with, we’ll ask visitors to
-                        book just one RMR slot each week.
+                        Materials Room.{' '}
+                        <strong>
+                          To start with, we’ll ask visitors to book just one RMR
+                          slot each week
+                        </strong>
+                        .
                       </li>
                       <li>
                         After you’ve booked your ticket, request up to 10 items
```

Comment:
What's the  rule on `.` on list items? Seems to be applied differently in different places.

Reply:
everywhere where lists are sentences we add stops (there's just one list where this isn't the case, I believe)

Reply:
Full sentences in bulleted lists get a `.`
Phrases in bulleted lists only get a `.` on the final bulleted phrase.

Reply:
Great! I've always wanted a rule for this in docs.

Reply:
![image](https://user-images.githubusercontent.com/1394592/93793883-505a9e80-fc2f-11ea-87bb-b67590df95ca.png)

…no final `.` after the final bulleted phrase in the doc from Kate

Reply:
Yes, I would.